docs/rest/Data/CustomSearchPatient.json

```diff
@@ -87,6 +87,7 @@
     "birthDate": "2016-01-15",
     "address": [
         {
+            "use": "usual",
```

Comment:
It looks like we need to pick the address use value from [this enum](http://hl7.org/fhir/valueset-address-use.html). I noticed we get a 400 error when we post this patient:

`"Error occurred when parsing model: 'While building a POCO: Literal 'usual' is not a valid value for enumeration 'AddressUse' (at Resource.address[0].use[0])'."`